tests/integration/test_pr_comment.py

```diff
@@ -334,7 +334,7 @@ def test_pr_comment_invalid(comment):
         " /packit build ",
         "asd\n/packit build\n",
         "asd\n /packit build \n",
-        "Should be fixed now, lets /packit build it.",
+        "Should be fixed now, let's\n /packit build\n it.",
```

Comment:
Maybe it's worth having a counter-example as well. (The one having command inside some text and/or the quoted command that triggered this change.)

Reply:
Added this entry:

    """comment with embedded /packit build not recognized
        unless /packit command is on line by itself""",

to the "comment" list that is input for the test_pr_comment_invalid() method.